src/config/default.toml

```diff
@@ -21,6 +21,7 @@ halt_afterwards = false
 restore_unwritten_bytes = false
 # The path where an SVG of the assembled flash layout should be written to.
 # flash_layout_output_path = "out.svg"
+do_chip_erase = false
```

Comment:
I think it would be good to add an explanatory comment, or at least a newline to separate it from the previous setting.

I first thought the comment above belonged to the `do_chip_erase` setting.

Reply:
Good point, I clarified this :)